crates/frontend/src/components/dropdown.rs

```diff
@@ -86,7 +97,8 @@ where
                                     type="text"
                                     class="grow"
                                     placeholder="Search"
-                                    name=name
+                                    ref_=node_ref
```

Comment:
wont this suffice ?
```suggestion
                                    autofocus={true}
```

Reply:
That wasn't working, so went ahead with this approach.